isaric/parsers/rapid_observations.toml

```diff
@@ -0,0 +1,782 @@
+[adtl.defs."admissionDateHierarchy"]
+combinedType = "firstNonNull"
+        fields = [
+            { field = "hostdat", description = "Admission date at this facility" },
+            { field = "cestdat", description = "Onset date of first/earliest symptom" },
+            { field = "dsstdat", description = "Date of enrolment" }
+        ]
+
+[adtl.defs."followupDateHierarchy"]
+combinedType = "firstNonNull"
+fields = [
+{ field = "flw_survey_date", description = "About you and your COVID-19 illness (if you're completing this survey on behalf of a child or adult that you care for, all the questions relate to their health and wellbeing)" },
+{ field = "flw2_survey_date", description = "About you and your COVID-19 illness (if you're completing this survey on behalf of a child or adult that you care for, all the questions relate to their health and wellbeing)" },
+{ field = "daily_date", description = "Date of follow-up" },
+{ field = "dsstdtc", description = "Outcome date" }
+]
+
+[[observation]]
+name = "haemorrhage"
+date = { ref = "admissionDateHierarchy" }
+phase = "admission"
+is_present = { field = "bleed_ceoccur_v2", ref = "Y/N/NK" }
+if = { bleed_ceoccur_v2 = { "!=" = 3 } }
+occurrence_period = 
+#description = "Bleeding (Haemorrhage)"
+
+[[observation]]
+name = "cough"
+date = { ref = "admissionDateHierarchy" }
+phase = "admission"
+is_present = { if field = "cough_ceoccur_v2", ref = "Y/N/NK" }
+if = { cough_ceoccur_v2 = { "!=" = 3 } }
+occurrence_period = 
+#description = "Cough"
+
+[[observation]]
+name = "cough"
+date = { ref = "admissionDateHierarchy" } 
+phase = "study"
+is_present = { if field = "daily_cough_ceoccur_v2", ref = "Y/N/NK" }
+if = { daily_cough_ceoccur_v2 = { "!=" = 3 } }
+occurrence_period = 
+#description = "Cough"
+
+[[observation]]
+name = "cough"
+date = { ref = "admissionDateHierarchy" } 
+phase = "admission"
+is_present = { if field = "symptoms_cough", values = { 1 = true, 0 = false } }
+if = { symptoms_cough = { "!=" = "" } }
+occurrence_period = 
+#description = "Cough"
+
+[[observation]]
+name = "avpu"
+date = { ref = "admissionDateHierarchy" }
+phase = "study"
+value_char = { field = "avpu_vsorres", values = { 1 = "Alert", 2 = "Verbal", 3 = "Pain", 4 = "Unresponsive" }}
```

Comment:
Should these two avpu be combined? Unless they are a different phase or occurrence_period

Reply:
there are mapping to two different fields.. one has 'daily_', which I imagine is a daily recording. We can discuss this further - whether to combine into one, or assume they have different occurrence period ..